test/testModel.cc

```diff
@@ -603,7 +603,8 @@ SUITE(Canvas)
         auto& group=dynamic_cast<Group&>(*itemFocus);
         group.updateBoundingBox();
         group.relZoom=0.5; // ensure displayContents is false
-        double w=group.iconWidth, h=group.iconHeight;
+        //double w=group.iconWidth, h=group.iconHeight;
```

Comment:
remove dead code.

Reply:
Done. 

The failure of some tests on my laptop may have something to do with the fact that it freezes up completely sometimes and then I have to restart it. It's doing it more often lately (it's really hot here at the moment, but I have a fan platform underneath the laptop). The Psensor temperature has been spiking to almost a 100 degrees C lately. I can't do anything else at the same time or else in the worst case it freezes up.

So, that may be why I'm seeing more failures, because there aren't enough resources available for some of the tests to run properly. I might run some diagnostics over the weekend.